receiver/signalfxreceiver/receiver.go

```diff
@@ -215,7 +214,7 @@ func (r *sfxReceiver) writeResponse(ctx context.Context, resp http.ResponseWrite
 	}
 
 	resp.WriteHeader(http.StatusOK)
-	resp.Write(okRespBody)
+	_, _ = resp.Write(okRespBody)
```

Comment:
Should we be checking that no error is returned or is it ok to ignore it?

Reply:
It's ok to ignore, there is no established way to report error in this function. At least we ignore it explicitly now

Reply:
I use failRequest function to handle this error, is that ok?